crates/ethereum-rpc/src/lib.rs

```diff
@@ -627,20 +635,41 @@ fn register_rpc_methods<C: sov_modules_api::Context, Da: DaService>(
                 let block =
                     evm.get_block_by_number(Some(BlockNumberOrTag::Latest), None, &mut working_set);
 
-                let synced_block_number = match block {
+                let l2_synced_block_number = match block {
                     Ok(Some(block)) => block.header.number.unwrap(),
                     Ok(None) => 0u64,
                     Err(e) => return Err(e),
                 };
 
-                if synced_block_number < head_block_number {
-                    Ok::<CitreaStatus, ErrorObjectOwned>(CitreaStatus::Syncing(SyncStatus {
-                        synced_block_number,
-                        head_block_number,
-                    }))
+                let l1_synced_block_number = match ethereum.ledger_db.get_last_scanned_l1_height() {
```

Comment:
for the sequencer this table is never updated. however we were already considering disabling healthcheck endpoint for sequencer. so do it

Reply:
I got confused about the healthcheck. this is a different rpc method. 

still sending this to the sequencer will be problematic

Reply:
and this is already only registered if not sequencer. cool